vector/src/main/AndroidManifest.xml

```diff
@@ -111,7 +112,9 @@
                 android:resource="@xml/shortcuts" />
         </activity-alias>
 
-        <activity android:name=".features.home.HomeActivity" />
+        <activity
+            android:name=".features.home.HomeActivity"
+            android:launchMode="singleTask" />
```

Comment:
I tried adding `singleTask` to this activity for a deeplink issue and it caused a bunch of odd behaviours (skipping deeplinks, activity not fully loading), have you noticed anything strange after the change?

Reply:
I didn't see any problem on HomeActivity after this change but I didn't stress it. I would prefer to make more tests.

Reply:
Indeed, clicking on a link like https://app.element.io/#/room/%23element-web-announcements:matrix.org

creates 2 tasks if Element was already running, so we can see in the task switcher:

<img width="309" alt="image" src="https://user-images.githubusercontent.com/3940906/142900245-ec74774a-e80a-497c-aa08-37f0ea44d250.png">

See https://github.com/vector-im/element-android/issues/2299 and the fix https://github.com/vector-im/element-android/pull/2341 :)

Reply:
is this a blocker for the fix? maybe `singleTop` could provide similar protection without allowing multiple instances

Reply:
No more needed to change launchMode since we generate a random task affinity on each build.